src/lib/constants/element/datepicker.py

```diff
@@ -0,0 +1 @@
+SELECTOR = "a.ui-state-default"
```

Comment:
For senity sake, these selectors could be in one file in classes and nested classes. It would be easier to search through and edit these values.